Source/WebCore/page/LocalFrame.h

```diff
@@ -223,7 +223,8 @@ class LocalFrame final : public Frame {
     WEBCORE_EXPORT float frameScaleFactor() const;
 
     void deviceOrPageScaleFactorChanged();
-    
+    void screenContentsFormatsChanged();
```

Comment:
The name is a bit abstract. Does a screen have a "contents format"? It's more like `screenSupportedContentsFormats` or `screenAppropriateContentsFormats`